.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "build-test"
+on:
+  push:
+    branches:
+      - net203
+  pull_request:
+    branches:
+      - net203
+
+jobs:
+  test-backend:
+    runs-on: ${{ matrix.image }}
+    strategy:
+      matrix:
+        image: [windows-latest]
+    env:
+      NUGET_PACKAGES: ${{ github.workspace }}/.github/nuget-packages
+      GRADLE_USER_HOME: ${{ github.workspace }}/.github/gradle
+    steps:
+      - uses: actions/checkout@v2
+
+      # Prepare
+      - name: Gradle JVM Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: build/gradle-jvm
+          key: ${{ runner.os }}.gradle-jvm.${{ hashFiles('rider-fsharp/gradle*') }}
+
+      - name: Gradle Wrapper Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.GRADLE_USER_HOME }}/wrapper
+          key: ${{ runner.os }}.gradle-wrapper.${{ hashFiles('rider-fsharp/gradle/**') }}
+
+      - name: NuGet Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.NUGET_PACKAGES }}
+          key: ${{ runner.os }}.nuget.${{ hashFiles('ReSharper.FSharp/**/*.csproj') }}+${{ hashFiles('ReSharper.FSharp/**/*.props') }}
+
+      - name: Gradlew Prepare
+        working-directory: ./rider-fsharp
+        run: ./gradlew prepare
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: '3.1.101'
+
+      - name: Add MSBuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      # Build and Test ReSharper.FSharp
+      - name: Test FSharp.Common.Tests
```

Comment:
Would it be better to have build and test as separate stages (so it's easier to see whether it's build or tests who's failing)?

Reply:
I think it will be much slower as ```dotnet test``` builds the project too

Reply:
Why would it be slower? Does it do a rebuild?

Reply:
First, if everything is set up correctly, pre-test build should be no-op.

Second, even if everything's not set up correctly, there's `dotnet test --no-build`.

Reply:
You can also `dotnet test` the solution file. That will test all projects in the solution, rather than you having to specify each test project. (It may also test the assemblies in parallel - not sure.)

Reply:
This would also split build warnings from test errors, which both are in the same task now, making finding test errors rather difficult: https://github.com/DedSec256/fsharp-support/pull/3/checks?check_run_id=1272374074